cardano-api/gen/Gen/Cardano/Api.hs

```diff
@@ -22,10 +22,10 @@ import qualified Hedgehog.Range as Range
 
 genMetadata :: MonadGen m => m (Metadata era)
 genMetadata = do
-  numberOfIndicies <- Gen.integral (Range.linear 1 15)
-  let indexes = map (\i -> fromIntegral i :: Word64) [1..numberOfIndicies]
-  mDatums <- Gen.list (Range.singleton numberOfIndicies) genMetadatum
-  return . Metadata . Map.fromList $ zip indexes mDatums
+  numberOfIndices <- Gen.integral (Range.linear 1 15)
+  let indices = map (\i -> fromIntegral i :: Word64) [1..numberOfIndices]
+  mData <- Gen.list (Range.singleton numberOfIndices) genMetadatum
+  return . Metadata . Map.fromList $ zip indices mData
```

Comment:
For example this should be it's own (micro) commit that just says it's fixing some spelling, so that we don't have to peer at it to see what is being changed or why.